src/components/Chapter/Chapter.js

```diff
@@ -20,7 +20,7 @@ export default function Chapter({ title, classes, onClose, type, reference }) {
   const [position, setPosition] = React.useState(initialPosition);
   const { state } = React.useContext(ResourcesContext);
   const {
-    state: { resourcesApp, fontSize },
+    state: {  resourcesApp, fontSize },
```

Comment:
Interesting, why prettier doesn't remove extra space?

Reply:
It's setting of my VScode at work. At home i fixed it.